src/map/script.c

```diff
@@ -26770,8 +26810,11 @@ static BUILDIN(mergeitem)
 	return true;
 }
 
-// getcalendartime(<day of month>, <day of week>{, <hour>{, <minute>}});
-// Returns the UNIX Timestamp of next ocurrency of given time
+/**
+ * Returns the UNIX Timestamp of next ocurrency of given time.
+ *
+ * getcalendartime(<day of month>, <day of week>{, <hour>{, <minute>{, <input_is_localtime>}}});
```

Comment:
oh, it looks like I wrote this comment totally wrong back then 🤦‍♂️ 

hour and minute are actually params 1 and 2. the script_commands is correct, but it is wrong here

```
getcalendartime(<hour>, <minute>{, <day of month>{, <day of week>{, <input_is_localtime>}}})
```

Reply:
Right, I didn't notice this